app/models/runtime/droplet_model.rb

```diff
@@ -138,7 +138,7 @@ def docker_user
         end
       end
 
-      container_user.presence || AppModel::DEFAULT_DOCKER_CONTAINER_USER
+      container_user.presence || (Config.config.get(:allow_docker_root_user) ? AppModel::DOCKER_CONTAINER_ROOT_USER : AppModel::DEFAULT_CONTAINER_USER)
```

Comment:
It took me a second to mentall parse this -- can we write it without the ternary? Same with the code below in the process model?